.eslintrc

```diff
@@ -275,7 +273,6 @@ rules:
   wrap-regex: 2
 
   # ECMAScript 6
-  arrow-parens: 2
```

Comment:
Gives false positives for async arrow functions. See https://github.com/eslint/eslint/issues/3354